qemu/tests/boot_with_max_mem.py

```diff
@@ -0,0 +1,25 @@
+from virttest import env_process, utils_misc
+
+
+def run(test, params, env):
+    """
+    Test steps:
+    1) Check whether the host memory is greater than boot guest memory:
+    If the host memory is greater than boot guest memory,
+    continue to test the case,Otherwise, skip the test case
+    2) Boot guest with max memory and two cpus
+    3) Check guest dmesg, no call trace message
+
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment.
+    """
+    params["start_vm"] = "yes"
+    vm_name = params["main_vm"]
+    env_process.preprocess_vm(test, params, env, vm_name)
+    vm = env.get_vm(vm_name)
+    vm.verify_alive()
+    timeout = float(params.get("login_timeout", 360))
+    session = vm.wait_for_login(timeout=timeout)
+    utils_misc.verify_dmesg(level_check=4, session=session)
```

Comment:
Suggest `vm.verify_dmesg`  and set suitable `guest_dmesg_level`

Reply:
The return result of verify_dmesg() is different from the expected result, so verify_dmesg() cannot be used, I will change the code in other ways, thank you